tests/dialects/test_hive.py

```diff
@@ -611,12 +611,6 @@ def test_hive(self):
                 "spark": "GET_JSON_OBJECT(x, '$.name')",
             },
         )
-        self.validate_all(
-            "STRUCT(a = b, c = d)",
-            read={
-                "snowflake": "OBJECT_CONSTRUCT(a, b, c, d)",
-            },
-        )
```

Comment:
why was this removed?

Reply:
didn't find any docs on hive struct

Reply:
added unnamed structs for hive, it doesn't look like it uses key value